com.zandronum.Zandronum.yaml

```diff
@@ -0,0 +1,161 @@
+app-id: com.zandronum.Zandronum
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "5.12"
```

Comment:
Update runtime to 5.13

Reply:
Wasn't aware that 5.13 is available. First time I heard of it. Anyway, I updated my package.